elements/pfe-layouts/pfe-base.css

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
why does this file appear in this MR?

Reply:
It was a leftover from migrating from compiling assets in root to compiling them into dist.  They no longer exist. :)